# 07 Maths Assignment 13th Sept

<img src="https://drive.google.com/uc?export=view&id=1qC2Egngjkqjpf7YaBEo72oBfd-A3io4P"  width="75%" height="75%">



\begin{equation}
Markov = 
\begin{bmatrix}
0.5 & 0.1 & 0.2\\
0.3 & 0.6 & 0.4\\
0.2 & 0.3 & 0.4
\end{bmatrix}
\end{equation}

We know that the Highest Eigen Value $\lambda$ is 1, which converges the Markov matrix towards the highest eigen value corresponding vector $V_{1}$
So our final probability, which the Markov matrix converges to after many simulations is $[v_1, v_2, v_3]$

$$
\begin{equation}
\begin{bmatrix}
0.5 & 0.1 & 0.2\\
0.3 & 0.6 & 0.4\\
0.2 & 0.3 & 0.4
\end{bmatrix}

\begin{bmatrix}
v_1 \\
v_2 \\
v_3
\end{bmatrix}

=

\begin{bmatrix}
v_1 \\
v_2 \\
v_3
\end{bmatrix}
\end{equation}
$$

Now we have
$$
\begin{equation}
    0.5v_1 + 0.1v_2 + 0.2v_3 = v_1 \tag{1}
\end{equation}
$$

$$
\begin{equation} 
    0.3v_1 + 0.6v_2 + 0.4v_3 = v_2 \tag{2}
\end{equation}
$$

$$
\begin{equation}
0.2v_1 + 0.3v_2 + 0.4v_3 = v_3 \tag{3}
\end{equation}
$$

Also, we know the values of the principal eigen vector, are probabilities of the states, so the sum of these values must be 1.



$$
\begin{equation}
    v_1 + v_2 + v_3 = 1 \tag{4}
\end{equation}
$$

Solving $(1)$, $(2)$ and $(4)$ we get

$$
v_1 = 0.2181
$$
$$
v_2 = 0.4727
$$
$$
v_3 = 0.3090
$$

Now, we start with 100 people at Hostel, 100 people at Plaksha and 100 people at Sector-17

In [ ]:
.2181*300, 0.4727*300, 0.3090*300

(65.42999999999999, 141.81, 92.7)

After the convergence, the state the system is in:

People at Hostel: $0.2181*300 = 65.5$

People at Plaksha: $0.4727*300 = 141.8$

People at Sector-17: $0.3090*300 = 92.7$
